lib/mobile_robot_simulator/robot_models/differential_robot.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python
+# Copyright (c) 2011 Autonomous Robots and Cognitive Systems Laboratory
+# Authors: Daniel Garcia Vaglio degv364@gmail.com
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+# Differential Robot Class.
+# This is a simple class that stores the parameters for a kinematic model
+# of a differential robot. Also computes the velocities that are necesary
+# to work with the simulator core
+
+from __future__ import division
+from builtins import object
+from past.utils import old_div
+from numpy import array, cos, sin
+
+
+class Differential_robot(object):
+    def __init__(self):
+        # dimensions
+        self.robot_dim = [.5, 0.5, .3]
+        self.robot_center = array([0., 0., 0])
+        self.robot_planes = [
+            # normal point
+            [(1., 0., 0), (old_div(self.robot_dim[0], 2.), 0., 0.)],
```

Comment:
old_div is not required here since this is a floating point division it will work consistently though python2 and python3. Please change to regular `/` division operator